apim/3.x/README.adoc

```diff
@@ -339,6 +339,10 @@ Please be aware that the Elasticsearch installed by Gravitee is NOT recommended
 `+null+`, defaults to Management API ingress value)_
 |`+[apim.example.com]/management+`
 
+|`+ui.scheme+` |Whether to use HTTP or HTTPS to communicate with Management API,
```

Comment:
```suggestion
```

Reply:
Just reading your latest commit, Do you need me to add docs to the readme file for?: 
- `api.ingress.management.scheme`
- `api.ingress.portal.scheme`